common/test/acceptance/tests/lms/test_certificate_web_view.py

```diff
@@ -88,3 +92,151 @@ def test_page_has_accomplishments_banner(self):
+            CourseUpdateDesc(date='January 29, 2014', content='Test course update1')
+        )
+
+        self.course_fixture.add_children(
+            XBlockFixtureDesc('static_tab', 'Test Static Tab'),
+            XBlockFixtureDesc('chapter', 'Test Section').add_children(
+                XBlockFixtureDesc('sequential', 'Test Subsection', grader_type='Final Exam').add_children(
+                    XBlockFixtureDesc('problem', 'Test Problem 1', data=load_data_str('multiple_choice.xml')),
+                    XBlockFixtureDesc('html', 'Test HTML'),
+                )
+            ),
+            XBlockFixtureDesc('chapter', 'Test Section 2').add_children(
+                XBlockFixtureDesc('sequential', 'Test Subsection 2', grader_type='Midterm Exam').add_children(
+                    XBlockFixtureDesc('problem', 'Test Problem 2', data=load_data_str('formula_problem.xml')),
+                )
+            )
+        )
+
+        self.course_fixture.install()
+        self.user_id = "99"  # we have created a user with this id in fixture
+        self.cert_fixture = CertificateConfigFixture(self.course_id, test_certificate_config)
+
+        self.course_info_page = CourseInfoPage(self.browser, self.course_id)
+        self.progress_page = ProgressPage(self.browser, self.course_id)
+        self.course_nav = CourseNavPage(self.browser)
+        self.tab_nav = TabNavPage(self.browser)
+
+    def log_in_as_unique_user(self):
+        """
+        Log in as a valid lms user.
+        """
+        AutoAuthPage(
+            self.browser,
+            username="testprogress",
+            email="progress@example.com",
+            password="testuser",
+            course_id=self.course_id
+        ).visit()
+
+    def test_progress_page_has_view_certificate_button(self):
+        """
+        Scenario: View Certificate option should be present on Course Progress menu if the user is
+        awarded a certificate.
+        And their should be no padding around the box containing certificate info. (See SOL-1196 for details on this)
+
+        As a Student
+        Given there is a course with certificate configuration
+        And I have passed the course and certificate is generated
+        When I go on the Progress tab for the course
+        Then I should see a 'View Certificate' button
+        And their should be no padding around Certificate info box.
+        """
+        self.cert_fixture.install()
+        self.log_in_as_unique_user()
+
+        self.complete_course_problems()
+
+        self.course_info_page.visit()
+        self.tab_nav.go_to_tab('Progress')
+
+        self.assertTrue(self.progress_page.q(css='.auto-cert-message').first.visible)
+
+        actual_padding = get_element_padding(self.progress_page, '.wrapper-msg.wrapper-auto-cert')
+        actual_padding = [int(padding) for padding in actual_padding.itervalues()]
+        expected_padding = [0, 0, 0, 0]
+
+        # See issue SOL-1196 for details of this check
```

Comment:
nit: We should rephrase this comment. It is open source project and referring a Jira ticket in code comment is not a good idea.